docs/sql-ref-syntax-ddl-create-table-datasource.md

```diff
@@ -118,7 +118,7 @@ CREATE TABLE student (id INT, name STRING, age INT) USING CSV;
 CREATE TABLE student_copy USING CSV
     AS SELECT * FROM student;
   
---Omit the USING clause, which uses the default data source (parquet by default)
+--Omit the USING clause, which uses the default data source (textfile by default)
```

Comment:
Why is this change? I thought `parquet` is the default data source (by `spark.sql.sources.default`).

Reply:
The hive table format depends on the conf hive.default.fileformat when set spark.sql.legacy.createHiveTableByDefault is true . And if spark.sql.legacy.createHiveTableByDefault is false, it hive table format depends on the conf spark.sql.sources.default . Now spark.sql.legacy.createHiveTableByDefault is true, and the table created by default is textfile.